crates/daphne-server/src/storage_proxy_connection/kv/cache.rs

```diff
@@ -29,44 +29,32 @@ pub struct Cache {
     kv: HashMap<&'static str, HashMap<String, CacheLine>>,
 }
 
-pub enum GetResult<T: 'static> {
-    NotFound {
-        /// Whether the caller should check KV for the value.
-        read_through: bool,
-    },
+pub enum CacheResult<T: 'static> {
+    Hit(Option<Marc<T>>),
+    Miss,
```

Comment:
`Hit` versus `Miss` is slightly confusing because `Hit(None)` is actually a miss as well. I would at least clarify in a comment that the caller is meant to skip reading through to KV if `None`.

Reply:
It's not a miss, the cache correctly cached that the value is not present, not present being a valid value for the given key

Reply:
Ah I see. I have a different understanding of what "Hit" is supposed to mean, so I do think a comment to clarify would help.

Reply:
Yes, this PR effectively wants to change the meaning of `Hit` to not have any special meaning, it means the wikipedia definition of a cache hit

> A cache hit occurs when the requested data can be found in a cache

hence why I believe it doesn't require a comment, my intent is that you can tell from the type that a `None` is a valid `Hit`